index.html

```diff
@@ -1,29 +1,125 @@
 <!DOCTYPE html>
 <html lang="en-ca">
   <head>
-    <!-- Set the page title -->
     <title>Assignment 02</title>
-
-    <!-- Set the charset we're using to UTF-8 -->
     <meta charset="UTF-8" />
-
-    <!-- Set the screens default zoom / scale to 1 -->
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    
-    <!-- Specifies the document mode for Internet Explorer -->
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-
-    <!-- We are using a CSS Reset file to standardize browser default css -->
     <link rel="stylesheet" href="reset.css" />
-
-    <!-- This is where all of our awesome CSS code goes -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lilita+One&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <!--
+    <header>
+        <div class="container">
+            <div class="logo"><img src="logo.png" img width="50"></div>
+            <div class="buttons">
+                    <nav>
+                      <ul>
+                        <button class="button1"><li><a href="Home.html">Home</a></li></button>
+                        <button class="button1"><li><a href="Shop.html">Shop</a></li></button>
+                        <button class="button1"><li><a href="Best.html">Best Sellers</a></li></button>
+                        <button class="button1"><li><a href="About.html">About</a></li></button>
+                        <button class="button1"><li><a href="#contactform">Contact</a></li></button>
+                      </ul>
+                    </nav>
+            </div>
+            <div class="search"><img src="search2.png" img width="40"></div>
+            <div class="profile"><img src="profile2.png" img width="40"></div>
+        </div>
+    </header>
 
-      YOU HAVE TO MODIFY THE HTML TO BUILD YOUR AWESOME PAGE
+    <div class="head">
+        <h1>Plush Central</h1>
+        <h3>Every Toy Has A Spirit of Joy</h3>
+    </div>
+        
+    <div class="arrivals">
+        <h1>New Arrivals</h1>
```

Comment:
This comment is informational: 

Ideally your page would just have one `h1` on it. 

You shouldn't have multiple `h1`s as it confuses screen readers and other assistive technologies